src/button/ux-button.ts

```diff
@@ -29,12 +29,24 @@ export class UxButton implements Themable {
     if (this.theme) {
       this.styleEngine.applyTheme(this, this.theme);
     }
+
+    if (this.disabled || this.disabled === '') {
+      this.button.setAttribute('disabled', '');
+    }
   }
 
   public themeChanged(newValue: any) {
     this.styleEngine.applyTheme(this, newValue);
   }
 
+  public disabledChanged(newValue: any) {

```

Comment:
Yup! Though in this case it'd be `boolean | string | null` as I believe an end user could technically null it out.